app/_src/networking/dns.md

```diff
@@ -119,8 +119,9 @@ Consuming a service handled by {{site.mesh_product_name}} DNS, whether from {{si
 <kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
 <kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
 ```
+Be aware of DNS standards ([RFC1035](https://www.ietf.org/rfc/rfc1035.txt)) if using underscores in the service name.
```

Comment:
I don't think this sentence brings any value, especially when we add the link to RFC to below section. If you are looking for DNS-compliant variant, there is assumption you are aware that your own service name has also follow the standard.
```suggestion
```